packages/fxa-payments-server/server/lib/routes/legal-docs.js

```diff
@@ -10,7 +10,6 @@ const fetch = require('node-fetch');
 const Sentry = require('@sentry/node');
 const { cdnFqdn, resultsCacheLimit } = config.get('legalDocLinks');
 const acceptLanuageParser = require('accept-language-parser');
-const supportedLanguages = require('fxa-shared/l10n/supportedLanguages.json');
```

Comment:
1. What is the intended purpose of the `supportedLanguages` list?
From what I can ascertain, the idea is that we add a language to this list when the majority of strings have been localized in Pontoon.

2. Why remove this constraint altogether instead of adding `ms` to the list of `supportedLocales`?
The Malay locale in Pontoon is only 31% translated, so it would probably not be considered a supported language in this case.

3. I think this bug may highlight the larger issue of the fact that we don’t really have sufficient l10n translations for Malay?
I.e. Does this mean that it’s possible for a user in an "unsupported locale" to have other FxA content rendered in a different (supported) locale?E.g. A locale of `ms` would have no matches, and we would fall back to the default English for subscription sign-up, since `supportedLocales` is used by `AppLocalizationProvider`?